pkg/server/grpc/grpc.go

```diff
@@ -67,17 +73,40 @@ func New(logger log.Logger, reg prometheus.Registerer, tracer opentracing.Tracer
 		return status.Errorf(codes.Internal, "%s", p)
 	}
 
+	loggingOpts := []grpc_logging.Option{
+		grpc_logging.WithDecider(func(_ string) grpc_logging.Decision {
+			return LogDecision[requestLoggingDecision]
```

Comment:
Do we need this Log map?

Reply:
Yeah, this one is for the logging levels option we use from the flags

Reply:
Can we change this

```
// Decision defines rules for enabling start and end of logging.
type Decision int

const (
	// NoLogCall - Logging is disabled.
	NoLogCall Decision = iota
	// LogFinishCall - Only finish logs of request is enabled.
	LogFinishCall
	// LogStartAndFinishCall - Logging of start and end of request is enabled.
	LogStartAndFinishCall
)

```